### ISSUE TITLE
Accessibility -> Cannot navigate to the Left navigation menu on A&R using the "Tab" key/Keyboard

### ISSUE THREAD

**Issue by kgorana** [bcgov/bcparks-ar-api#261]
## Description
**How to replicate:**
* Navigate to the A&R test environment: [A&R Admin](https://d3ahrzu0mpndj3.cloudfront.net/)
* Try to navigate to the left navigation menu using the keyboard tab key

**Expected Behavior:**
* User should be able to access all the website components using the keyboard

**Issue:**
* cannot access the left navigation menu using the Tab key

## Linked issues
relates to
- [x] ~[BRS-822](https://bcparksdigital.atlassian.net/browse/BRS-822)~
